File: Python3/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    pnum=0
    ynum=0
    # initialize before using
    
    for i in s:
        if i.lower()=='p':
            #lower()
            pnum+=1
        elif i.lower()=='y':
            ynum+=1
            
    if pnum!=ynum:
        answer=False
    return answer

File: Java/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int pnum=0;
        int ynum=0;
        for(int i=0;i<s.length();i++){
            if(s.charAt(i)=='p' || s.charAt(i)=='P'){
                //charAt(i)
                pnum+=1;
            }
            else if(s.charAt(i)=='y' || s.charAt(i)=='Y'){
                ynum+=1;
            }
        }
        
        if(pnum!=ynum){
            answer=false;
        }
        return answer;
    }
}